container/Containerfile

```diff
@@ -0,0 +1,233 @@
+FROM quay.io/centos/centos:stream8
+
+# The name of the Ceph release
+ARG CEPH_VERSION_NAME="main"
+
+# (optional) Choose a specific Ceph package version to install
+# This value is appended directly to ceph packages to select their version
+# e.g., "-18.2.0"
+ARG CEPH_PACKAGE_VERSION=""
+
+# (optional) Choose a specific NFS-Ganesha package version to install
+# This value is appended directly to nfs-ganesha packages to select their version
+# e.g., "-5.5-1.el8s"
+ARG GANESHA_PACKAGE_VERSION=""
+
+# Set this to "true" to force usage of Shaman for devel builds
+ARG IS_DEVEL="false"
+
+# Used as URL "ref" for selecting Shaman packages
+ARG CEPH_REF="${CEPH_VERSION_NAME}"
+
+# Set to "crimson" if needed
+ARG OSD_FLAVOR="default"
+
+
+# TODO: Is this still used? Can we get rid of this? @adk3798 do you know?
+#       ref: https://github.com/ceph/ceph-container/pull/1604
+# Is a ceph container ?
+LABEL ceph="True"
+
+# TODO: This doesn't seem to be used by rook or ceph. Can we get rid of this? @guits do you know?
+#       ref: https://github.com/ceph/ceph-container/pull/1969
+ENV I_AM_IN_A_CONTAINER 1
+
```

Comment:
@adk3798 @guits do you see a need to keep these?

Reply:
Cephadm doesn't use these directly. It might link somehow to the CEPH_USE_RANDOM_NONCE stuff though. Cephadm's use of that was just recently removed in https://github.com/ceph/ceph/pull/50344

Reply:
In a search of the ceph repo, I didn't find any references to I_AM_IN_A_CONTAINER, but I'm not sure if there could be usages I'd miss by searching like that.

Reply:
That might have been just for humans and/or custom scripting in the container.  It's good to be able to query, although maybe there's some more-standard way?...

Reply:
can a process running inside a container query the container labels?